package_mrtrix

```diff
@@ -72,7 +64,7 @@ EOF
 
     QTCORE=$(for n in $LIBS; do echo $n; done | grep 'libQt.*Core')
     [[ $QTCORE ]] && ( # sort out Qt:
-      QTPLUGDIR=$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
+      QTPLUGDIR=/usr/lib/qt/plugins/ #$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
```

Comment:
I believe `/usr/lib/qt/plugins/` is `/usr/lib/x86_64-linux-gnu/qt5/plugins/` on ubuntu? QTCORE is /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 and $(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p') yields `/usr/lib/x86_64-linux-gnu/qt5/plugins`.